### PR TITLE
Feat: Upgrade to 0.15

### PR DESCRIPTION
**Pull request by Meldiron**
- [x] Manual QA

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (lohanidamodar) [inline .vscode/settings.json]: Why?

**Review by lohanidamodar — APPROVED** [review]
Just a comment

File: .vscode/settings.json (hunk 1 above)
Comment (Meldiron) [reply .vscode/settings.json]: for VS code, to give proper linting, checking, all of that stuff. We didn't have it, and after adding it, it showed like 50 errors.

Not sure if we want to keep it, I would say yes.

Comment (Meldiron) [reply .vscode/settings.json]: Removed the file, it's VS code specific so let's not keep it.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM denoland/deno:alpine-1.10.3
+FROM denoland/deno:alpine-1.23.1
```
Comment (Meldiron) [inline Dockerfile:1]: TODO:
- [ ] Rename buckets back to storage
- [ ] Remove datababase-related stuff from collection, and move it to database/index.ts

File: .vscode/settings.json (hunk 1 above)
Comment (stnguyen90) [reply .vscode/settings.json]: Personally, I like having things like this in the source code to help ensure consistency for multiple developers working on the code base.

**Comment by loks0n** [comment]
closing in favour of #16

**Pull request closed without merging**